### DISCUSSION TITLE
Sur l'outil statistique #2

### DISCUSSION THREAD

**Discussion by gauthierandre** [betagouv/ma-cantine#1165] [Idées]
Suite à une question utilisateur, est-ce qu'on pourrait afficher la proportion entre établissement publics et privés sur cette page : https://ma-cantine.beta.gouv.fr/statistiques-regionales ? Qu'en pensez-vous @hfroot @alemangui ? ou peut-être qu'elle est quelque part, et je ne l'ai pas vue.

**Comment by hfroot** [comment]
Oui, j'imagine ça serait facile d'ajouter

**Reply by hfroot** [reply]
J'ai créé l'issu https://github.com/betagouv/ma-cantine/issues/1289
